apps/web/src/pages/quick-start/consts.tsx

```diff
@@ -195,3 +205,17 @@ export const frameworkInstructions: { key: string; value: ISnippetInstructions[]
     ],
   },
 ];
+
+export enum OnBoardingAnalyticsEnum {
+  FRAMEWORK_SETUP_VISIT = '<FRAMEWORK> Setup Page Visit',
+  FRAMEWORKS_SETUP_VISIT = 'Framework Setup Page Visit',
+  OTHER_CHANNELS_VISIT = 'Other Channels Page Visit',
+  NOTIFICATION_CENTER_SETUP_VISIT = 'Notification Center Setup Page Visit',
+  QUICK_START_VISIT = 'Quick Start Page Visit',
+  TRIGGER_VISIT = 'Trigger Page Visit',
+  CLICKED_FAQ = 'Clicked On FAQ',
+  CLICKED_CREATE_TEMPLATE = 'Clicked On Create Template',
+  CLICKED_TRIGGER_EVENT = 'Clicked On Trigger Event',
+  EXCEPTION = 'Exception was thrown',
+  COPIED_STEP = 'Copied Step <STEP>',
```

Comment:
Same here, pass the step type as a property

Reply:
i guess you refer only to COPIED_STEP right?

Reply:
@scopsy